tests/test_vonkarman.py

```diff
@@ -25,19 +25,13 @@
 
 
 @timer
-def test_vk(slow=False):
+def test_vk():
     """Test the generation of VonKarman profiles
     """
-    if slow:
-        lams = [300.0, 500.0, 1100.0]
-        r0_500s = [0.05, 0.15, 0.3]
-        L0s = [1e10, 25.0, 10.0]
-        do_deltas = [False, True]
-    else:
-        lams = [500.0]
-        r0_500s = [0.2]
-        L0s = [25.0]
-        do_deltas = [False]
+    lams = [300.0, 500.0, 1100.0]
+    r0_500s = [0.05, 0.15, 0.3]
+    L0s = [1e10, 25.0, 10.0]
+    do_deltas = [False, True]
```

Comment:
How slow were the slow tests?  I'm assuming you checked this and found the "slow" ones were still fast enough to run all the time.

Reply:
On my lap this this is 2.07 sec for all tests vs 0.07 sec for just the fast tests.  I can wrap this in a `if __name__ == '__main__'` block if you prefer.

Reply:
Sure.  I usually try to keep each test under 1 sec if possible for they pytest run.  There are some exceptions, but this seems like one where we don't need the slow tests to spot regressions.